### PR TITLE
Typo Fixes

### PR DESCRIPTION
**Pull request by ashok-hariharan**
Fixing two issues as part of my first contribution to Open Source 
1. Added unit test policy #832  (will raise a separate PR for this issue as it needs more details)
2. Typo in multiple files #989 

Please check this @robmoffat

### CODE REVIEW

File: CONTRIBUTING.md
Comment (kriswest) [inline CONTRIBUTING.md:86]: Couple of types to fix:

```suggestion
- Any change to the FDC3 specification's typescript codebase must be supported by accompanying unit tests.
```

Comment (kriswest) [inline CONTRIBUTING.md:86]: Some more detail would also be useful here - FDC3 is made up of 4 specifications, one of which relates to the Desktop Agent API. Most of the codebase is just typing to define that API standard. However, there is actual code in method.ts, which is currently all that we apply unit tests to. There is also code in ContextTypes.ts, which is auto-generated from context schemas. This does not currently include tests - and we might not want to be forced to add them when new Context type schemas are added and start autogeneratingnew functions.

There are other apps in the /toolbox directory. These don't currently have unit tests.
We might add further apps there, e.g. a basic app directory implementation - the existing out-of-date implementation was recently removed.

**Review by kriswest — CHANGES_REQUESTED** [review]
Type fixes all look good.

We need a bit more thought on/detail in the unit test policy before merging that. Hence, you might consider splitting this into two PRs so that they can be handled separately.

File: CONTRIBUTING.md
Comment (ashok-hariharan) [reply CONTRIBUTING.md]: Removed unit test policy changes. Will keep it as a separate PR  by adding some more additional details to it

**Review by kriswest — APPROVED** [review]
LGTM! Will update title to reflect removal of Unit test policy

**Review by greyseer256 — APPROVED** [review]

**Comment by ashok-hariharan** [comment]
I don't have write access, requesting @greyseer256 or @kriswest to merge it

**Pull request merged**
